ovos_audio/playback.py

```
@@ -117,6 +129,16 @@
             data, message.context = self.tts_transform.transform(data, message.context)
 
             self.p = play_audio(data)
+
+            if not visemes and self.g2p is not None:
+                try:
+                    visemes = self.g2p.utterance2visemes(message.data["utterance"],
+                                                        get_message_lang(message))
+                except OutOfVocabulary:
+                    pass
+                except:
```

Ruff

Do not use bare `except`

(E722)